raiden/ui/cli.py

```diff
@@ -311,19 +324,22 @@ def run(ctx, **kwargs):
             raiden_api = RaidenAPI(app_.raiden)
             rest_api = RestAPI(raiden_api)
             api_server = APIServer(rest_api, cors_domain_list=domain_list)
+            (api_host, api_port) = split_endpoint(kwargs["api_address"])
 
             Greenlet.spawn(
                 api_server.run,
-                ctx.params['api_port'],
+                api_host,
+                api_port,
                 debug=False,
                 use_evalex=False
             )
 
             print(
-                "The RPC server is now running at http://localhost:{}/.\n\n"
+                "The RPC server is now running at http://{}:{}/.\n\n"
```

Comment:
@andrevmatos Since you as a new user of Raiden also got confused by this message I suggest you change it in your PR.

Perhaps change the beginning of the sentence to: "The Raiden API RPC server ..."